util.py

```diff
@@ -8,14 +8,14 @@
 
 RESPONSE_REACTIONS = ["🇦", "🇧", "🇨", "🇩"]
 
-async def prompt_for_turn(bot: commands.Bot, user: discord.User, battlecontext) -> Turn:
+async def prompt_for_turn(bot: commands.Bot, user: discord.User, BattleContext) -> Turn:
```

Comment:
avoid name conflict
```suggestion
async def prompt_for_turn(bot: commands.Bot, user: discord.User, battlecontext: BattleContext) -> Turn:
```

Reply:
this will also require the other references to this variable to be changed

Reply:
got it. thanks